Update tv.ir

Synthethics opened the pull request:
# What's new

Added VOX Electronics Model LED 43ADS316B TV Remote.

# Verification 

Just pop it in an IR File and try it with said TV. I can also record it works as I use it on my own TV. I've also replace the original tv.ir file with my the commited one and all of the buttons on the universal remote function on the TV.

# Checklist (For Reviewer)

- [x] PR has description of feature/bug or link to Confluence/Jira task
- [x] Description contains actions to verify feature/bugfix
- [x] I've built this code, uploaded it to the device and verified feature/bugfix

Synthethics committed:
Update tv.ir

Added VOX Electronics LED 43ADS316B TV Remote.

Synthethics committed:
Update tv.ir

Synthethics committed:
Merge branch 'dev' into Synthethics-patch-1

skotopes committed:
Merge branch 'dev' into Synthethics-patch-1

skotopes commented on assets/resources/infrared/assets/tv.ir:
```diff
@@ -1717,3 +1717,63 @@ type: raw
+address: 40 00 00 00
+command: 12 00 00 00
+# 
+name: Output
```
Please carefully read https://github.com/flipperdevices/flipperzero-firmware/blob/dev/documentation/UniversalRemotes.md

Only `Power`, `Mute`, `Vol_up`, `Vol_dn`, `Ch_next`, and `Ch_prev` must be included

skotopes commented:
Please un-draft when all issues are fixed

Synthethics committed:
Updated tv.ir

Fixed and removed unnecessary buttons to fit with IR file standards as per https://github.com/flipperdevices/flipperzero-firmware/blob/dev/documentation/UniversalRemotes.md

skotopes committed:
Merge remote-tracking branch 'origin/dev' into Synthethics-patch-1

skotopes committed:
Assets: reorder and tag new tv remote

skotopes committed:
Assets: cleanup infrared universal tv
